server/channels/store/sqlstore/channel_store.go

```diff
@@ -2020,6 +2020,83 @@ func (s SqlChannelStore) GetChannelMembersTimezones(channelId string) ([]model.S
 	return dbMembersTimezone, nil
 }
 
+type channelsWithUnreadsAndWithMentionsQueryResult struct {
```

Comment:
Slight preference towards declaring this as an anonymous struct inside the method. No need to pollute the global namespace with it. 0/5.